css/default.css

```diff
@@ -92,3 +92,10 @@ td.print-overdue    { font-weight: bold; }
 .table-nonfluid {
    width: auto !important;
 }
+
+/* Markdown */
+.markdown-body blockquote {
```

Comment:
We should support markdown everywhere where the following APIs are called by MantisBT core or plugin:

- string_display
- string_display_line
- string_display_links
- string_display_line_links
- string_rss_links

If so, then it is pretty much anywhere within the MantisBT emitted content.

Let's see what others think of the approach as well.